pkg/config/dynamicconfig.go

```diff
@@ -195,13 +195,21 @@ func (cd *ConfigData) updateCM(old, cur interface{}) {
 		return
 	}
 	// if data has not changed then dont load configmap
-	reconcilePolicyReport, updateWebook := cd.load(*cm)
-	if reconcilePolicyReport {
+	reconcilePolicyReport, updateWebhook := cd.load(*cm)
+
+	cd.ReconcilePolicyReport(reconcilePolicyReport)
+	cd.UpdateWebhook(updateWebhook)
+}
+
+func (cd *ConfigData) ReconcilePolicyReport(reconcile bool) {
```

Comment:
No we don't, thanks for mentioning this. I had earlier implemented it in a way where it was always sending the reconcile signal regardless of `true` or `false`, but then I realised there's no need to send a signal in case of `false`. Forgot to remove the param, will fix this asap!